_events/2022/2022-11-sc22.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Research Software Engineers in HPC Workshop (RSE-HPC-2022)
+location: Dallas, TX
+url: https://us-rse.org/rse-hpc-2022/
+expires: 2022-11-13
+event_date: "November 13, 2022"
+layout: event
+category: workshop
+time:
+    - - start: 2022-11-13T20:00:00Z
+        end: 2022-11-13T23:30:00Z
+---
+
+We are excited to announce the Research Software Engineers in HPC Workshop (RSE-HPC-2022)
+to be held as part of [SC22](https://sc22.supercomputing.org/) (hybrid event, in Dallas, TX and virtually).
+This will be a half-day workshop 2:00pm-5:30pm CST (UTC-6), Sunday, November 13, 2022.
+
+This workshop will bring together RSEs and allies involved in HPC, from all over the world,
+to grow the RSE community by establishing and strengthening professional networks of current
+RSEs and RSE leaders.  We’ll discuss the current activities and plans of national RSE
+organizations, discuss the needs of RSEs and RSE groups, and brainstorm ways RSE organizations
+can help address these.
+
+We encourage prospective participants to submit position papers (limit of 2 pages,
+not counting references; no format prescribed) on topics related to RSE issues.
+Papers and discussion topics should be submitted using the [submission website](https://submissions.supercomputing.org/?page=Submit&id=SC22WorkshopRSEHPC2022Submission&site=sc22).
+The deadline for submissions is Friday, August 5, 2022, at 11:59 PM AoE (UTC-12).
+
+For more information, see the [workshop website](https://us-rse.org/rse-hpc-2022/)
+or [contact the organizers](mailto: sc-ws-rse-hpc@info.supercomputing.org).
```

Comment:
```suggestion
or [contact the organizers](mailto:sc-ws-rse-hpc@info.supercomputing.org).
```

Reply:
I just made this change

Reply:
Dan, you were too fast. I noticed a typo (extra space) and went to edit my comment. But you'd already changed it :).